solr/solrj/src/java/org/apache/solr/client/solrj/request/DirectXmlRequest.java

```diff
@@ -20,13 +20,15 @@
 import org.apache.solr.client.solrj.request.RequestWriter.StringPayloadContentWriter;
 import org.apache.solr.client.solrj.response.UpdateResponse;
 import org.apache.solr.client.solrj.util.ClientUtils;
+import org.apache.solr.common.params.ModifiableSolrParams;
 import org.apache.solr.common.params.SolrParams;
 
 /**
  * Send arbitrary XML to a request handler
  *
  * @since solr 1.3
  */
+@Deprecated
```

Comment:
There's a lot I want to deprecate and remove in Solr 10; this is one.  The annotation snuck in; arguably for another PR.  On the other hand, Solr isn't using or testing this class, so let's get on with removing!

Reply:
I want to consolidate deprecations elsewhere so I will remove this change from the PR.  We can discuss when I post that.